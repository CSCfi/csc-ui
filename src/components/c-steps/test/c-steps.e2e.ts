import { newE2EPage } from '@stencil/core/testing';
import { CSteps } from '../c-steps';

describe('c-steps', () => {
  it('renders default html', async () => {
    const page = await newE2EPage();
    await page.setContent(`<c-steps></c-steps>`);

    expect(page).toMatchSnapshot();
  });
});
