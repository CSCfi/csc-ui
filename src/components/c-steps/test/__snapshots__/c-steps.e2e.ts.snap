// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`c-steps renders default html 1`] = `undefined`;
